src/gui/systray.h

```diff
@@ -54,25 +54,18 @@ class Systray
     void showMessage(const QString &title, const QString &message, MessageIcon icon = Information);
     void setToolTip(const QString &tip);
     bool isOpen();
-    Qt::WindowFlags windowFlags() const;
-    qreal windowRadius() const;
-    qreal windowWidth() const;
-    qreal windowHeight() const;
     QString windowTitle() const;
+    bool normalWindow() const;
```

Comment:
should not this be called `useNormalWindow` ?

Reply:
Not sure if thats better, but I will change it.